src/content/en/fundamentals/security/csp/index.md

```diff
@@ -301,7 +301,7 @@ out-of-line, but that's a tradeoff that's well worth making.
 ### If you absolutely must use it ...
 
 CSP Level 2 offers backward compatibility for inline scripts by allowing you to
-whitelist specific inline scripts using either a cryptographic nonce (number
+allowlist specific inline scripts using either a cryptographic nonce (number
```

Comment:
"allowing you to allowlist" sounds awkward. Maybe "by letting you allowlist" or "by letting you add specific inline scripts ... to the allowlist"?

Reply:
I change it to "by allowing you to add specific inline scripts to the allowlist"

Reply:
Hmm, the repetition of "allow" is what's awkward IMHO.